app/customers.py

```diff
@@ -0,0 +1,118 @@
+import json
+import math
+
+from app.shop import Shop
+
+
+class Customer:
+
+    def __init__(
+            self,
+            name: str,
+            product_cart: dict,
+            location: list[int],
+            money: int,
+            car: dict
+    ) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
+        self.money = money
+        self.car = car
+
+    with open("app/config.json", "r") as file:
+        data = json.load(file)
+    fuel_price = data["FUEL_PRICE"]
+
+    outskirts_shop = Shop(data["shops"][0]["name"],
+                          data["shops"][0]["location"],
+                          data["shops"][0]["products"])
+
+    shop_24 = Shop(data["shops"][1]["name"],
+                   data["shops"][1]["location"],
+                   data["shops"][1]["products"])
+
+    central_shop = Shop(data["shops"][2]["name"],
+                        data["shops"][2]["location"],
+                        data["shops"][2]["products"])
```

Comment:
The code exhibits redundancy as it manually creates Customer instances for each customer data entry from the JSON file.